.github/workflows/publish-docker-explorer.yml

```diff
@@ -1,40 +1,40 @@
-name: Build and publish docker image
+name: Build and publish explorer docker image
 
 on:
   push:
-    tags: [ "v[0-9]+.[0-9]+.[0-9]+*" ]
+    tags: ['alephium-explorer@[0-9]+.[0-9]+.[0-9]+*']
 
 jobs:
-  buildx_and_push_to_registry:
+  buildx_and_publish_to_registry:
     name: Push Docker image to Docker Hub
     runs-on: ubuntu-latest
     steps:
-      - name: Check out the repo
-        uses: actions/checkout@v2
+      - name: Check out Git repository
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
           ref: ${{ github.event.workflow_run.head_branch }}
 
-      - name: Setup node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: 16
+      - name: Install Bun package manager
+        uses: oven-sh/setup-bun@v1
+
+      - name: Install NPM dependencies
+        run: bun install
 
       - name: Build the static website
-        run: npm ci && npm run build
+        run: cd apps/explorer && bun run ci:build
 
-      - uses: docker/setup-qemu-action@v1
-      - uses: docker/setup-buildx-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
       - name: Get the version
         id: get_version
-        run: |
-          version=$(git describe --tags --abbrev=0)
-          echo $version
-          echo ${version:1}
-          echo ::set-output name=VERSION::$version
-          echo ::set-output name=VERSION-NO-V::${version:1}
-        shell: bash
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
+        with:
+          path: apps/explorer
```

Comment:
Now that the tags are prefixed with the package name (still discussing this on Slack), instead of figuring out how to extract the version from the tag, I decided to use an action that gets the version from the package.json file.

Reply:
Makes sense.